ui/pages/Staff/components/Users.jsx

```diff
@@ -29,15 +40,15 @@ const COLUMNS = [
 
 const getUserFilterVal = ({ email, displayName }) => `${email}-${displayName}`
 
-const Users = React.memo(({ users, loading, load }) =>
+const Users = React.memo(({ users, loading, load, hijak }) =>
   <DataLoader load={load} content loading={false}>
     <DataTable
       striped
       idField="username"
       defaultSortColumn="email"
       loading={loading}
       data={users}
-      columns={COLUMNS}
+      columns={hijak ? HIJAK_COLUMNS : COLUMNS}
```

Comment:
Should it show HIJAK_COLUMNS only when the user is a hijak-authorized user?

Reply:
this is a staff-only page, and hijak only shows during local development which is just staff users, so this is fine